### DISCUSSION TITLE
A alarm system panel

### DISCUSSION THREAD

**Discussion by MathieuMQc** [MiguelRipoll23/homebridge-securitysystem#299] [Ideas]
Hey guys ! Just an idea that would be awesome, if you could have a website that we could control the alarm system from a tablet. I think it is already possible if we do our own but i'm not a programmer and I don't know how I could do it... Thanks you !!!
A website would be the best I think because we could use any VERY old tablet.

**Comment by rvaneeuwijk** [comment]
I have solved that by using VIZ Designer for HomeKit to design and run an app that runs on a tablet. I control the security system with that. And I am no programmer either.
